src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/ViewModelBase.cs

```diff
@@ -11,5 +12,17 @@ public void RaisePropertyChanged([CallerMemberName] string propertyName = "")
         {
             PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
         }
+
+        public bool SetAndRaisePropertyChanged<T>(ref T field, T newValue, [CallerMemberName] string propertyName = "")
```

Comment:
The callers to `SetAndRaisePropertyChanged` are not relying on the `bool` return value. I this method can return `void`. Please let me know if I am missing something.

Reply:
Hmm @donnie-msft  this was your suggestion. That said I think returning bool informs the caller if the property was actually changed. While I'm not doing anything with that information here, I can see that being useful for another view model.

Reply:
The way it's implemented in this PR is the standard approach.